src/ScoutingMain.elm

```diff
@@ -67,20 +72,30 @@ stylishPage position title teamNumber page =
             , Font.color Colors.veryLightBlue
             , Font.size size
             ]
+
+        findColor : String -> Element.Color
+        findColor alliance =
+            if String.contains "Blue" alliance then
+                Colors.blue
+
+            else if String.contains "Red" alliance then
+                Colors.red
+
+            else
+                Colors.blue
     in
     column
-        [ Background.color blue
-        , padding 105
+        [ Background.color <| findColor station
```

Comment:
Why is station a `String`?
This means you can get a `station` like `""` or `"not a real station"` or `"you get the point"` and so on.
You have exactly 6 possible stations, 1-3, blue or red.
How about the following encoding
```elm
type Alliance
    = Red
    | Blue


type DriverStationNumber
    = One
    | Two
    | Three


type alias DriverStation = ( Alliance, DriverStationNumber )
```
this way you cannot create an invalid `DriverStation` and can easily translate any `DriverStation` to its color or number.